src/features/creators/autoFindProjects.ts

```diff
@@ -54,10 +54,12 @@ export class AutoFindProjects implements PythonProjectCreator {
     public readonly displayName = ProjectCreatorString.autoFindProjects;
     public readonly description = ProjectCreatorString.autoFindProjectsDescription;
 
+    supportsQuickCreate = true;
+
     constructor(private readonly pm: PythonProjectManager) {}
 
     async create(_options?: PythonProjectCreatorOptions): Promise<PythonProject | PythonProject[] | undefined> {
-        const files = await findFiles('**/{pyproject.toml,setup.py}');
+        const files = await findFiles('**/{pyproject.toml,setup.py}', '**/.venv/**');
```

Comment:
Actually, we need to use the Prefix path for the environment. This is not always good enough. I think for now `.venv` is ok, but really we should use the sysPrefix path for environment for the workspace and ignore all `pyproject.toml` or `setup.py` from there.